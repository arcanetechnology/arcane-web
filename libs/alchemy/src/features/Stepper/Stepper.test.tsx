/** @format */

import { Stepper, StepperTab } from '.';
import { cleanup, render } from 'solid-testing-library';

describe('alchemy stepper element', () => {
  afterEach(cleanup);
  it('should be able to render form element and functional children', () => {
    const tree = render(() => (
      <Stepper>
        <StepperTab>
          <h1>Hello</h1>
        </StepperTab>
        <StepperTab>
          <h1>world</h1>
        </StepperTab>
      </Stepper>
    )).baseElement;

    expect(tree).toMatchSnapshot();
  });
});
