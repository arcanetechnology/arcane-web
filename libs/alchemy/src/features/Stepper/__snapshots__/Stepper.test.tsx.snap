// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alchemy stepper element should be able to render form element and functional children 1`] = `
<body>
  <div>
    <form>
      <div
        id="stepper-cl-0-tab"
      >
        <h1>
          Hello
        </h1>
      </div>
      <div
        id="stepper-cl-0-tab"
      >
        <h1>
          world
        </h1>
      </div>
    </form>
    <div
      id="stapper-cl-0-actions"
    >
      <button
        class="elevation-200 border-small radius-large"
        id="arcane-standard-button"
      >
        Back
      </button>
      <button
        class="elevation-200 border-small radius-large"
        id="arcane-standard-button"
      >
        Next
      </button>
    </div>
  </div>
</body>
`;
